aten/src/ATen/native/cpu/UpSampleKernelAVXAntialias.h

```diff
@@ -71,6 +71,8 @@ void pack_rgb(
     const at::Tensor& unpacked_tensor, // IN
     const at::Tensor& packed_tensor // OUT
 ) {
+  // Convert back RGBA into RGB.
```

Comment:
I wonder if we should rename those helpers (and slightly change this comment), because they can also handle C==1 and C==2 (not just C==3 so they're not just relevant for RGB tensors).

Reply:
maybe `unpack_to_4C` and `pack_from_4C`?